server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -105,9 +107,36 @@ func (s *BuildBuddyServer) GetInvocation(ctx context.Context, req *inpb.GetInvoc
 		return nil, err
 	}
 
+	if *paginateInvocations {
```

Comment:
if paginateInvocations is set, we should not call `LookupInvocation` anymore

Reply:
that's my plan with the TODO - just want to get this smaller PR submitted first